radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/sync/BFTSync.java

```diff
@@ -369,13 +349,14 @@ private void processGetVerticesLocalTimeout(VertexRequestTimeout timeout) {
 		}
 
 		if (syncRequestState.authors.isEmpty()) {
+			//TODO: potentially may cause node failure. Issue should be handled without throwing exception
```

Comment:
This is also just a sanity check and exception is fine.

Reply:
done